app/components/fullNode/FullNodeLoadingPage.js

```diff
@@ -0,0 +1,146 @@
+// @flow
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { resetNodeSettings } from '/redux/fullNode/actions';
+import {
+  PageWrapper,
+  LeftPane,
+  RightPane,
+  RightHeaderText,
+  BaseImage,
+  RightPaneInner,
+  LeftPaneInner,
+  ActionLink,
+  RightHeaderWrapper,
+  ImageWrapper,
+  LoadingRow,
+  ItemTextWrapperUniformPadding,
+  BottomLinksWrapper,
+  CenterTextWrapper,
+  LinkTextWrapper,
+  ItemText,
+  FullNodeHeaderText
+} from './FullNodeJointStyles';
+import { thinking } from '/assets/images';
+import LoadingBar from './LoadingBar';
+import { SmButton } from '/basicComponents';
+import type { DropdownEntry } from '/basicComponents';
+import FullNodeLog from './FullNodeLog';
+import FullNodeStatus from './FullNodeStatus';
+
+type SeupPageProps = {
+  history: any,
+  resetNodeSettings: Function,
+  fullNode: any
+};
+type SetupPageState = {
+  drive: ?DropdownEntry,
+  capacity: ?DropdownEntry
+};
+
+const rightPaneLoadingModeLinks = [
+  {
+    id: 'learnMore',
+    text: 'Learn more about Spacemesh Full Node'
+  },
+  {
+    id: 'showComputerEffort',
+    text: 'Show computer effort'
+  }
+];
+
+const loadingInformationText = "Full node setup may take up to 48 hours. you can leave it running and continue using your device as usual, just don't turn it off.";
+
+class FullNodeLoadingPage extends Component<SeupPageProps, SetupPageState> {
+  render() {
+    return [
+      <div>
+        <FullNodeHeaderText>Full Node Setup</FullNodeHeaderText>
+      </div>,
+      <PageWrapper>
+        <LeftPane>{this.renderLeftPane()}</LeftPane>
+        <RightPane>{this.renderRightPane()}</RightPane>
+      </PageWrapper>
+    ];
+  }
+
+  renderRightPane = () => (
+    <RightPaneInner>
+      <ImageWrapper>
```

Comment:
turn this to basic component Image that accepts only max-height param so the image can be proportional
<ImageWrapper>
        <BaseImage src={thinking} width={132} height={152} />
      </ImageWrapper>